modules/custom/movie_reservation_controller/templates/article-list.html.twig

```diff
@@ -1,14 +1,160 @@
 <h4>{{ title}}</h4>
 
-<ul>
-    {%  for article in items %}
+{{ attach_library('movie_type_filter_controller/form-styling') }}
+
+
+<div class="popup" id="popup">
+    <h1 style="text-align: center;">Select date for movie</h1>
+    
+    <button type="button" class="cancel_btn" id="cancel" onclick="cancelPopup()">Cancel</button>
+
+</div>
+
+<form>
+  <label for="movie_type">Please select a movie genre for which you would like to make a reservation:</label>
+      <select id="movie_type">
+
+            {%  for item in items %}
+                <option value="{{ item.id }}">{{ drupal_entity('taxonomy_term', item.id) }}</option>
+            {% endfor %}
+    
+        </select>
         
-        <li>
-             {{ drupal_entity('node', article.id ) }}
-        </li>
-    {% endfor %}
-</ul>
+<button type="button" id="submit">Choose movie type</button>
+
+
+</form>
+
+<div class="movies_for_type" style="margin-top:50px;">
+
+  <h1>We have found thease movies for the specified type</h1>
+
+  <form id="movie_form">
+
+
+  </form>
+
+  <button type="button" id="reserve" onclick="openPopup()">Reserve movie</button>
 
+</div>
 
 
+
+<script
+  src="https://code.jquery.com/jquery-3.6.0.js"
+  integrity="sha256-H+K7U5CnXl1h5ywQfKtSj8PCmoN9aaq30gDh27Xc0jk="
+  crossorigin="anonymous"></script>
+
+
+<script>
+
+  let movie_form = $("#movie_form");
+
+  let movies_for_type = $( ".movies_for_type" ).hide();
+
+  let popup = $("#popup").hide();
   
+  let reserve = $("#reserve").hide();
+
+  let table = document.getElementById('di-locations');
+
+  $( "#submit" ).click(function(e) {
+    e.preventDefault();
+
+    let movie_type = $("#movie_type").val();
+
+    $.ajax({
+      method:'GET',
+      url:'http://drupal.praksa/movie/type/'+movie_type,
+      success:function(response)
+      {
+       let resData = response["data"];
+
+       movie_form.html("");
```

Comment:
Da li ovo ima za zadatak da svaki put ocisti prethodni html forme?
Ako je to, onda ima jedna, rekao bih bolja opcija, a to je mozda .empty(), pokusaj sa time.